apps/aecore/lib/aecore/naming/tx/name_claim_tx.ex

```diff
@@ -42,8 +42,9 @@ defmodule Aecore.Naming.Tx.NameClaimTx do
 
   # Callbacks
 
+  # TODO integrate Identifiers
```

Comment:
is this todo still to be done?

Reply:
This is just a part of unused comments, will remove them soon.